[EDU-4623] fix: adjusting API Key field description for Elasticsearch endpoints - Data Stream (#1481)
aziontech/docs: src/content/docs/en/pages/observe-journey/data-stream/integrations/connector-elasticsearch.mdx

1. On the **Connector** dropdown menu, select **Elasticsearch**.
2. On **URL**, add the URL of your endpoint you copied on Elasticsearch. Example: `https://my-deploy-pg4by0.es.us-central1.gcp.cloud.es.io`

> 3. On **Encoded API Key**, add the `Base64` key corresponding to the `"encoded"` value generated during the creation of the API key in ElasticSearch. Example: `VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==`

Author: ```suggestion
3. On **Encoded API Key**, add the `Base64` key corresponding to the `"encoded"` value generated while creating the API key in ElasticSearch. Example: `VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==`
```

1. On the **Endpoint Type** dropdown menu, select **Elasticsearch**.
2. On **Elasticsearch URL**, add the URL of your endpoint you copied on Elasticsearch. Example: `https://my-deploy-pg4by0.es.us-central1.gcp.cloud.es.io`

> 3. On **Encoded API Key**, add the `Base64` key corresponding to the `"encoded"` value generated during the creation of the API key in ElasticSearch. Example: `VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==`

Author: ```suggestion
3. On **Encoded API Key**, add the `Base64` key corresponding to the `"encoded"` value generated while creating the API key in ElasticSearch. Example: `VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==`
```
